libs-sol/nearbridge/contracts/NearBridge.sol

```diff
@@ -31,11 +31,13 @@ contract NearBridge is INearBridge {
         mapping(uint256 => BlockProducer) next_bps;
     }
 
-    uint256 constant public LOCK_ETH_AMOUNT = 1 ether;
-//    uint256 constant public LOCK_DURATION = 1 hours;
-    uint256 constant public LOCK_DURATION = 10 seconds;
+    uint256 constant public DEFAULT_LOCK_ETH_AMOUNT = 1 ether;
+//    uint256 constant public DEFAULT_LOCK_DURATION = 1 hours;
+    uint256 constant public DEFAULT_LOCK_DURATION = 10 seconds;
 
     bool public initialized;
+    uint256 public lock_eth_amount;
```

Comment:
Is it in Eth or Wei?

Reply:
it's in wei, common practice in solidity is amount always in wei, and use ether is suffixed by the unit: `1 ether`, `ether` is then compiled to `* 1e18`